manim/animation/creation.py

```diff
@@ -400,9 +402,9 @@ def construct(self):
     def __init__(
         self,
         vmobject: VMobject,
-        rate_func: Callable[[float], float] = linear,
```

Comment:
Should `rate_func` be passed to `super`? (i.e. in this case `Write`). It is currently unused.